### PR TITLE
Feat/AN-3227 transform files in types folder to typesript

### PR DESCRIPTION
**Pull request by MGJamJam**
Issue: https://bitmovin.atlassian.net/browse/AN-3227

Work:
- installs `grunt-ts` to be able to compile typescript files
- transforms `.js` files in `types/ `folder to typescript
- cleans up code and extracts utility functions into own `utils/` folder

### CODE REVIEW

File: src/requestHandler.ts [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class RequestHandler {
+  backendService: any;
```
Comment (MGJamJam) [inline src/requestHandler.ts:2]: Transforming this file into TypeScript was necessary to maintain folder structure with grunt-ts compilation. The right types for this file (and all other missing JavaScript files) will be addressed in a separate PR.

File: src/utils/queryUtils.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import {AD_ATTRIBUTE, ATTRIBUTE} from "../types/queryAttributes";
+
+export const isNullFilter = (filter): boolean => {
```
Comment (sabe25) [inline src/utils/queryUtils.ts:3]: Can we give the parameter `filter` a type?

**Review by sabe25 — APPROVED** [review]
There is one small request on my side about the filter type 
Otherwise LGTM 👍
